postinstall.js

```diff
@@ -6,17 +6,22 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 'use strict';
+const path = require('path');
+const fs = require('fs');
+
+let notNPM = true;
+const setup_file = path.resolve(__dirname, './setup.js');
+try {
+  fs.accessSync(setup_file);
+} catch (e) {
+  notNPM = false;
+}
+
+const execute = require('./execute');
 
 if (
-  process.platform !== 'win32' &&
-  process.env.NODE_ENV !== 'production'
+  notNPM ||
+  process.env.NODE_ENV === 'production'
```

Comment:
I left it in so you can globally set NODE_ENV to production to skip it in case of issues, also for debugging you can just 
`NODE_ENV=production npm install`